.github/workflows/validate-flusight-testhub-config.yaml

```
@@ -68,7 +72,7 @@ jobs:
       - name: "Comment on PR"
         id: comment-diff
         if: ${{ github.event_name != 'workflow_dispatch' }}
-        uses: carpentries/actions/comment-diff@main
+        uses: carpentries/actions/comment-diff@e27aa6c531dadd357d2aa4c9a21e90849e23e963  #v0.14.0
```

This will not be the correct release. In fact, this will cause errors because this version used node 16 while GHA runners need node 20.

see https://github.com/carpentries/actions/pull/91

```suggestion
        uses: carpentries/actions/comment-diff@5d73d6a34b013488264890868d8eeab1edffdf2e
```

The release of v0.14.0 was before the panic set in and I just needed things to _work_. I'm going to contact them and see if I can get them to bump the version.

---

https://github.com/carpentries/actions/issues/102

---

Great catch--thank you! Do you have a sense of how responsive the team is to these kinds of requests? If it will take a while, I could switch this action back to `@main` and create a ticket to followup.

---

They've been running a skeleton crew since the beginning of 2024, so I think reverting to main and creating a ticket followup would make sense.